src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/DefaultServiceAccountProvider.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.workbench.leonardo.auth
+
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.workbench.leonardo.model.ServiceAccountProvider
+import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * A ServiceAccountProvider which uses all defaults.
+  */
+class DefaultServiceAccountProvider(config: Config) extends ServiceAccountProvider(config) {
```

Comment:
A trivial `ServiceAccountProvider` impl. This is referenced in the Leo example conf.